Vostok.Applications.AspNetCore/Middlewares/UnhandledErrorMiddleware.cs

```
@@ -1,19 +1,35 @@
 ﻿using System;
 using System.Threading.Tasks;
+using JetBrains.Annotations;
 using Microsoft.AspNetCore.Connections;
 using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Options;
+using Vostok.Applications.AspNetCore.Configuration;
 using Vostok.Logging.Abstractions;
 
 namespace Vostok.Applications.AspNetCore.Middlewares
 {
-    internal class UnhandledErrorMiddleware : IMiddleware
+    /// <summary>
+    /// Catches and logs unhandled exception on low level. Upon catching one, serves an error response.
+    /// </summary>
+    [PublicAPI]
+    public class UnhandledErrorMiddleware
```

May be `UnhandledExceptionMiddleware`? The HTTP errors are not handled here